README.md

```diff
@@ -137,7 +141,128 @@ function debug(plugins) {
 }
 ```
 
-##### `lint`
+#### `encapsulate`
+
+_(experimental)_
+
+* Type: `Boolean`
+* Default: `false`
+
+Resets CSS properties to their [initial values](https://developer.mozilla.org/en-US/docs/Web/CSS/initial_value)
+to effectively allow a component to opt out of CSS inheritance and be
+encapsulated from the rest of the application similar to [the Shadow DOM](https://www.w3.org/TR/shadow-dom/).
+There are two types of CSS properties that affect components, inherited (e.g.
+`font-size,` `color`) and non-inherited (e.g. `margin`, `background`). This
+option works so that:
+
+* Root elements (e.g. `.Component`) have both inherited and non-inherited
+  properties reset to default values.
+* Descendants (e.g. `.Component-item`) only have non-inherited properties reset
+  as this allows properties set on the root element to be inherited by its
+  descendants.
+
+This means that components are isolated from styles outside the component root
+element but should an inheritable property such as `font-size` be applied on the
+component root element it will be inherited by the component descendants as
+normal. This prevents the need to redeclare properties on every descendant in a
+component.
+
+The same rules also apply to nested components.
+
+**Rationale**
+
+One of the difficulties with CSS components is predictably. Unwanted styles
+can be inherited from parent components and this can make it difficult to
+reuse components in different contexts.
+
+Methodologies such as SUIT and BEM exist to try and mitigate this somewhat but
```

Comment:
I would clarify that SUIT, BEM, CSS Modules etc try to solve the cascade (global selector and specificity war) issue. This feature instead solves the second half of the problem: unpredictable inheritance.